packages/teleport/src/services/user/makeAcl.ts

```diff
@@ -30,6 +30,7 @@ export default function makeAcl(json): Acl {
   const tokens = json.tokens || defaultAccess;
   const accessRequests = json.accessRequests || defaultAccess;
   const billing = json.billing || defaultAccess;
+  const dbServers = json.dbServers || defaultAccess;
```

Comment:
we need to test this. we always try to test our api responses, so if we make a change to an existing `make*`, there is usually a test for it, if not, we make one